src/Select.js

```diff
@@ -320,6 +323,11 @@ const Select = React.createClass({
 		}
 	},
 
+	handleRequired (value, multi) {
+		if (!value) return true;
+		return (multi ? value.length === 0 : Object.keys(value).length === 0);
```

Comment:
multi value is an array, while single value is an Object like `{ label: 'Whatevz', value: 'Bananas' }`

I guess there is one case I may have missed, and that is when `simpleValue === true`. If I remember correctly, in that situation value can be a number or string.

If you create a bug please ping me and I'll be happy to look into that.